packages/block-editor/src/components/link-control/style.scss

```diff
@@ -135,6 +135,11 @@ $block-editor-link-control-number-of-actions: 1;
 		background-color: $light-gray-300;
 	}
 
+	// The added specificity is needed to override.
+	&:focus:not(:disabled) {
+		box-shadow: 0 0 0 $border-width-focus $theme-color inset;
```

Comment:
Where is the original rule defined?

Reply:
https://github.com/WordPress/gutenberg/blob/6675dc56560dd4741fec374a174c143e988fe24c/packages/components/src/button/style.scss#L33

Reply:
Should it also be `inset` there?